### ISSUE TITLE
Submission of TP

### ISSUE THREAD

**Issue by kennethSty** [nus-cs2113-AY2425S1/forum#47]
Is one person per team enough to submit the DG, UG, and jar?

**Comment by okkhoy** [comment]
yes! it is made as a team submission. we take the latest one even if multiple members submit
